cmd/prometheus/main.go

```diff
@@ -229,6 +229,10 @@ func main() {
 		"Maximum duration compacted blocks may span. For use in testing. (Defaults to 10% of the retention period.)").
 		Hidden().PlaceHolder("<duration>").SetValue(&cfg.tsdb.MaxBlockDuration)
 
+	a.Flag("storage.tsdb.max-chunk-size",
```

Comment:
```suggestion
	a.Flag("storage.tsdb.max-chunk-segment-size",
```

Reply:
I would explicitly calling out blocks, we have m-mapped chunks too now. How about `storage.tsdb.max-block-chunk-segment-size`?